### ISSUE TITLE
Create Employee list screen

### ISSUE THREAD

**Issue by smangan360** [Furlough-Friends/rota-flex-101#58]
**Requires #50 to be finished before starting**

- Create employees component.
- Call endpoint to return all active employees
- Create table and render all active employees in this table
- Create add button (doesn't do anything) 
- Each employee should have a delete and edit option with the correct icons as per image below. The functionality of these buttons are out of scope for this tickey
- Style table with consistency with the rest of the application and mock idea. Make it look "nice". Without UX there is a lot left up to developer in terms of styling decisions. 

**NOTE: Full time is derived from an employee with 37.5 contracted hours**

<img width="663" alt="Initial mock" src="https://user-images.githubusercontent.com/54022972/79991949-c4d7b880-84aa-11ea-8c8f-8a50b22d9e26.PNG">

**Comment by QuarkNerd** [comment]
Will we be having this component make the api call directly? Or are we going through redux? I ask because Im used to the latter but also used to the redux side of things being explicitly mentioned so thought id ask.

**Comment by emab** [comment]
> Will we be having this component make the api call directly? Or are we going through redux? I ask because Im used to the latter but also used to the redux side of things being explicitly mentioned so thought id ask.

Would that involve using a thunk? That's what I've done before. We could start with it in this component, and then move to a thunk later if other components need access to the same api call?

**Comment by QuarkNerd** [comment]
> > Will we be having this component make the api call directly? Or are we going through redux? I ask because Im used to the latter but also used to the redux side of things being explicitly mentioned so thought id ask.
> 
> Would that involve using a thunk? That's what I've done before. We could start with it in this component, and then move to a thunk later if other components need access to the same api call?

Thunk, or a saga.  Yeah it might make sense to wait.
